src/main/java/bsh/util/JConsole.java

```diff
@@ -820,6 +819,26 @@ public void setWaitFeedback( boolean on ) {
 
     private int textLength() { return text.getDocument().getLength(); }
 
+    @Override
```

Comment:
here I haven't taken out the streams yet. let's finalize Interpreter first, than we can come back to this one.